prisma/schema.prisma

```diff
@@ -105,3 +106,23 @@ model Attachment {
   @@index([filePath], name: "IX_Attachments_filePath")
   @@map("Attachments")
 }
+
+model ActivityLog {
+  id           String       @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  createdAt    DateTime     @default(now())
+  taskId       String       @db.Uuid
+  workspaceId  String       @db.VarChar(32)
+  activityType ActivityType
+  details      Json         @db.JsonB
+  task         Task         @relation(fields: [taskId], references: [id])
+  deletedAt    DateTime?
+
+  @@index([createdAt])
+  @@map("ActivityLogs")
+}
+
+enum ActivityType {
+  CREATE_TASK
+  ASSIGN_TASK
+  WORKFLOWSTATE_UPDATE
+}
```

Comment:
I think we are missing activity type for update description

Reply:
I haven't tracked logs for both title and description changes because they are not present in the Figma design. Let me know your thoughts on this. Should we confirm with the client first? @rrojan

Reply:
Confirmed and not needed for now! @rrojan